# motion type: Forward, Backward, Left, Right, CW, CCW, Stop, Recover, ForwardLeft, ForwardRight
string motion

# linear velocity
float32 velocity

# angular rate
float32 rate